design/README.md

```diff
@@ -0,0 +1,15 @@
+# ROS2 Navigation Design documenation
```

Comment:
Inconsistent capitalization. I'd say either "ROS2 Navigation design documentation" or "ROS2 Navigation Design Documentation" Should be consistent on titles.